### ISSUE TITLE
Algorithm

### ISSUE THREAD

**Issue by patrickbochen** [MadcowD/ALRNeuralNetwork#45]
Implement second order Newton's method for training neural network.  Add accelerative learning rate to that algorithm. Test the effects of the newly designed algorithm
